sql/ShardSet.go

```diff
@@ -42,12 +43,13 @@ func WithLogger(logger *log.Logger) func(*ShardSet) {
 		s.log = logger
 	}
 }
-func NewShardSet(username string, password string, host string, port int, dbNamePrefix string, numShards int, maxShards int, maxConns int, options ...func(*ShardSet)) *ShardSet {
+func NewShardSet(username string, password string, host string, port int, tls string, dbNamePrefix string, numShards int, maxShards int, maxConns int, options ...func(*ShardSet)) *ShardSet {
```

Comment:
I would prefer if this was a functional option rather than a new [required] function parameter, because otherwise it breaks backward compatibility.

Reply:
mm great idea. then we don't need this: https://github.com/ASAPPinc/ASAPP/pull/4862